.eslintrc.js

```diff
@@ -6,5 +6,7 @@ module.exports = {
   },
   rules: {
     "unicorn/prefer-module": "off",
+    "unicorn/prefer-spread": "off",
+    "unicorn/prefer-number-properties": "off",
```

Comment:
And this ignores the linter in this repo, you should ignore these in base.js

Reply:
done